tests/test_callprofiler.py

```diff
@@ -221,7 +221,7 @@ def baz():
 
         # Check the number of lines
         stat_line_list = root_stat.split("\n")
-        self.assertEqual(len(stat_line_list), 7)
+        self.assertEqual(len(stat_line_list), 10)
```

Comment:
Because added the dividers, the lines of the table become 10 lines. I changed the lines from 7 to 10.